CRAYSAT-1495: Document limitations to `sat swap` blade replacement procedure (#2181)
Cray-HPE/docs-csm: operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System_Using_SAT.md

   `--dst-mapping` argument should be a mappings file of the MAC addresses, IP addresses, and component xnames from the new blade. If the new
   blade was removed from another system, the mappings file was saved while performing the
   [Removing a Liquid-cooled Blade from a System Using SAT](Removing_a_Liquid-cooled_blade_from_a_System_Using_SAT.md) procedure on the other system.

>    If the new blade is a brand new blade, (i.e. not from another running system,) then neither `--src-mapping` or `--dst-mapping` should be used.

Reviewer: ```suggestion
   If the blade is a new blade, (that is, not from another running system), do not use either `--src-mapping` or `--dst-mapping`.
```
